src/github/github.api.mock.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from './interfaces/user.interface';
+
+export class GithubApiMock {
+  /**
+   * Gets contributor info from github.
+   */
+  async getContributorInfo(contributorUsername: string): Promise<User> {
+    return {
+      username: contributorUsername,
+      name: 'User Name',
+      avatarUrl: 'https://username.com',
+    };
+  }
```

Comment:
```suggestion
 getContributorInfo(contributorUsername: string): Promise<User> {
    return Promise.resolve({
      username: contributorUsername,
      name: 'User Name',
      avatarUrl: 'https://username.com',
    });
  }
```

c'est plus propre de ne pas mettre de `async` si on n'a pas besoin spécifiquement d'un `await` dans le code